[2022-11-25T09:19:15.215+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Postgres_Data.create_table scheduled__2022-11-24T00:00:00+00:00 [queued]>
[2022-11-25T09:19:15.272+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Postgres_Data.create_table scheduled__2022-11-24T00:00:00+00:00 [queued]>
[2022-11-25T09:19:15.274+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-25T09:19:15.277+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2022-11-25T09:19:15.281+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-25T09:19:15.344+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2022-11-24 00:00:00+00:00
[2022-11-25T09:19:15.369+0000] {standard_task_runner.py:55} INFO - Started process 52 to run task
[2022-11-25T09:19:15.386+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Postgres_Data', 'create_table', 'scheduled__2022-11-24T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/ConnectionTable.py', '--cfg-path', '/tmp/tmpcg5qbdqh']
[2022-11-25T09:19:15.390+0000] {standard_task_runner.py:83} INFO - Job 43: Subtask create_table
[2022-11-25T09:19:15.455+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-11-25T09:19:15.707+0000] {task_command.py:376} INFO - Running <TaskInstance: Postgres_Data.create_table scheduled__2022-11-24T00:00:00+00:00 [running]> on host 89ee96b17a7e
[2022-11-25T09:19:15.972+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=klein
AIRFLOW_CTX_DAG_ID=Postgres_Data
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-11-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-24T00:00:00+00:00
[2022-11-25T09:19:16.004+0000] {base.py:71} INFO - Using connection ID 'DB_ytb' for task execution.
[2022-11-25T09:19:16.077+0000] {sql.py:315} INFO - Running statement: CREATE TABLE IF NOT EXISTS COMMENTS (
    authorDisplayName VARCHAR NOT NULL,
    textDisplay VARCHAR NOT NULL
);, parameters: None
[2022-11-25T09:19:16.093+0000] {postgres.py:96} INFO - NOTICE:  relation "comments" already exists, skipping

[2022-11-25T09:19:16.129+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Postgres_Data, task_id=create_table, execution_date=20221124T000000, start_date=20221125T091915, end_date=20221125T091916
[2022-11-25T09:19:16.211+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-25T09:19:16.333+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
